packages/components/docs/public.md

```diff
@@ -34,11 +34,16 @@ You can then import `@labkey/components` in your application as follows:
 import { QueryModel, GridPanel } from '@labkey/components';
 ```
 
+#### API
+* [QueryModel API and GridPanel](./QueryModel.md) - how to use QueryModel as a selectRows API wrapper and display data
```

Comment:
Should we change the title/label here to include DetailPanel now that it is documented in the QueryModel.md file? So something like "QueryModel API (using GridPanel and/or DetailPanel)"?